src/main/java/fi/helsinki/cs/tmc/core/communication/ExerciseDownloader.java

```diff
@@ -194,8 +179,8 @@ public double getPercents(int exCount, int exercisesSize) {
      * @param zipUrl url which will be downloaded
      * @param path where to download
      */
-    private void downloadExerciseZip(String zipUrl, String path) {
-        File file = new File(path);
+    private void downloadExerciseZip(String zipUrl, Path path) {
+        File file = new File(path.toString());
```

Comment:
Is it possible to replace this `File` with `Path` as well? It's a bit out of scope of this issue so if it's non-trivial we can generate a new issue for this migration.